mapproxy/util/ext/wmsparse/duration.py

```diff
@@ -0,0 +1,586 @@
+"""

```

Comment:
mapproxy/util/ext is for vendorized code. wmsparse is an external package by me (but never published). It would be a good idea to move this package out of ext/ eventually  (but not in this PR) as there is no external source to pull fixes from.

For this reason, the package should not be extended. How about mapproxy/util/ext/isodate.py?